src/components/formElements/ButtonSign.vue

```diff
@@ -2,10 +2,7 @@
 export default {
   props: {
     styleType: String,
-    big: {
-      type: Boolean,
-      default: false,
-    },
+    size: String,
```

Comment:
let's use in any of that type of props validation function to explicitly declare the values which it can take